user/database-setup.md

```diff
@@ -131,6 +131,23 @@ before_install:
   - mysql -e "create database IF NOT EXISTS test;" -uroot
 ```
 
+### MySQL 5.6
+
+The recommended way to get MySQL 5.6 is switching to our [Trusty CI Environment](/user/trusty-ci-environment/) and manually install the required packages by adding the following lines to the `.travis.yml`:
+
+```yaml
+dist: trusty
+sudo: required
+addons:
+  apt:
+    packages:
+    - mysql-server-5.6
+    - mysql-client-core-5.6
+    - mysql-client-5.6
+```
+
+Note that the default root password will be set by the package. You can set it to travis (which is the value used everywhere else), do: ...
+
```

Comment:
As well as adding info on how to set the root password, are we also telling people what the default is set to? Or is it generated each time?